src/statsig/constants.ts

```diff
@@ -41,16 +51,16 @@ export const DynamicConfigs = {
   [StatsigMultiNetworkDynamicConfig.MULTI_CHAIN_FEATURES]: {
     configName: StatsigMultiNetworkDynamicConfig.MULTI_CHAIN_FEATURES,
     defaultValues: {
-      showCico: [networkConfig.defaultNetworkId],
-      showBalances: [networkConfig.defaultNetworkId],
-      showSend: [networkConfig.defaultNetworkId],
-      showSwap: [networkConfig.defaultNetworkId],
-      showTransfers: [networkConfig.defaultNetworkId],
-      showWalletConnect: [networkConfig.defaultNetworkId],
-      showApprovalTxsInHomefeed: [] as NetworkId[],
-      showNfts: [networkConfig.defaultNetworkId],
-      showPositions: [networkConfig.defaultNetworkId],
-      showShortcuts: [networkConfig.defaultNetworkId],
+      showCico: defaultNetworkIds,
+      showBalances: defaultNetworkIds,
+      showSend: defaultNetworkIds,
+      showSwap: defaultNetworkIds,
+      showTransfers: defaultNetworkIds,
+      showWalletConnect: defaultNetworkIds,
+      showApprovalTxsInHomefeed: celoAndEthereumNetworkIds,
```

Comment:
I think we should do all here too.. idk why we even have this limitation currently in Valora

Reply:
i asked in slack https://valora-app.slack.com/archives/C025V1D6F3J/p1736467016275029